administrator/language/en-GB/plg_schemaorg_organization.ini

```
@@ -0,0 +1,19 @@
+; Joomla! Project
+; (C) 2023 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_SCHEMAORG_ORGANIZATION="Schema.org - Organization"
+PLG_SCHEMAORG_ORGANIZATION_DESCRIPTION_LABEL="A Schema.org Type <a href=\"https://schema.org/Organization\" target=\"_blank\">More information</a>"

```

```suggestion
PLG_SCHEMAORG_ORGANIZATION_DESCRIPTION_LABEL="A Schema.org Type <a href=\"https://schema.org/Organization\" target=\"_blank\"  rel=\"noopener noreferrer\">More information</a>"

```

This is for security purposes. Sorry your mentors didnt inform you of this